packages/headless/src/features/triggers/trigger-actions.ts

```diff
@@ -0,0 +1,6 @@
+import {createAction} from '@reduxjs/toolkit';
+
+/**
+ * Clears the triggers query parameter to an empty string.
+ */
+export const clearQueryTrigger = createAction('triggers/queryClear');
```

Comment:
What is this action for?

Reply:
Yes, I just had a chat with Jean Felix. This action is not needed and I will rework the controller's functionality instead of using this action.